### ISSUE TITLE
Using Quora's ?share=1 param

### ISSUE THREAD

**Issue by drusepth** [ismnoiet/anonymous-quora-browsing#1]
Not sure if you're aware, but Quora has a `?share=1` param you can append to pages to supposedly remove the signup wall, and might be pretty easy to inject into page params on load instead of mucking with the DOM (which is prone to Quora updating classes, structure, etc). 

Is there a benefit you get over direct DOM manipulation that appending `?share=1` to a page doesn't get you?

**Comment by ismnoiet** [comment]
Hi @drusepth thanks for reaching out and letting me know about this trick and 
sorry for responding very late.
To be honest I didn't know about that, now I tried it and it works.
If I came across this solution earlier I wouldn't touch the DOM and using CSS selectors(which may change over time).
I believe this solution it is the right way to do it and I'll implement it very soon.
Once again thanks for your visit and valuable feedback, the OSS community really needs people like you.
